geode-core/src/main/java/org/apache/geode/management/internal/i18n/CliStrings.java

```diff
@@ -1914,9 +1921,10 @@
   public static final String REMOVE__MSG__KEY_EMPTY = "Key is Null";
   public static final String REMOVE__MSG__REGION_NOT_FOUND = "Region <{0}> Not Found";
   public static final String REMOVE__MSG__KEY_NOT_FOUND_REGION = "Key is not present in the region";
-  public static final String REMOVE__MSG__CLEARED_ALL_CLEARS = "Cleared all keys in the region";
-  public static final String REMOVE__MSG__CLEARALL_NOT_SUPPORTED_FOR_PARTITIONREGION =
-      "Option --" + REMOVE__ALL + " is not supported on partitioned region";
+  public static final String REMOVE__MSG__CLEARED_ALL_KEYS = "Cleared all keys in the region";
```

Comment:
It might be best to rename and move this constant to reflect that going forward, only the clear command should be using the message, since remove --all is now deprecated.

Reply:
I agree, this could easily be corrected here.

Reply:
I still think that renaming this to something like "CLEAR_REGION_CLEARED_ALL_KEYS" and moving it to the "clear region" section of constants would be worthwhile. It seems like a good idea to keep all clear-related stuff in one place.

Reply:
Done.